main/get.c

```diff
@@ -829,7 +830,8 @@ extern int cppGetc (void)
 				int next = getcFromInputFile ();
 
 				if (next == NEWLINE)
-					continue;
+					// continue; <-- this may exit returning the backslash, which si wrong.
```

Comment:
This is already covered by an existing test which turned out to produce wrong type name output.
Units/readtags.r/backslash-at-the-end-of-pattern.d produced type \int.

How do I remove a single file change from a commit and then readd it as separate?